### PR TITLE
Added logger

### LINKED ISSUE #9: Implement logging
We are have a oslo.log for logging. Let's make it more useful

### PR DESCRIPTION
**Pull request by esikachev**
Added logger for my-dev-server project.
Close #9

### CODE REVIEW

File: my_dev_server/db/base.py [hunk 1]
```diff
@@ -4,14 +4,20 @@
 from sqlalchemy.ext import declarative as dec
 
 from my_dev_server import config
+from my_dev_server import logger
```
Comment (zemuvier) [inline my_dev_server/db/base.py]: Maybe something like 
 _from my_dev_server import config, logger_

File: my_dev_server/logger.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
+import functools
+import logging
+import os
+import traceback
+
+from oslo_config import cfg
+from my_dev_server import config
```
Comment (zemuvier) [inline my_dev_server/logger.py]: Add empty line between 6 and 7

**Comment by zemuvier** [comment]
Aslo fix error on ci with permission denied

File: my_dev_server/db/base.py (hunk 1 above)
Comment (esikachev) [inline my_dev_server/db/base.py]: take a look on our HACKING

**Comment by esikachev** [comment]
@zemuvier @IlyaMenkov please, review

File: my_dev_server/logger.py [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+                    '{func} raised: {exc!r}\n'
+                    'Traceback: {tb!s}'.format(
+                        func=func.__name__, exc=e, tb=traceback.format_exc()))
+                raise
+            return result
+        return wrapped
+    return wrapper
+
+
+logwrap = debug(logger)
```
Comment (zemuvier) [inline my_dev_server/logger.py]: Do we need the empty line in the end of file?

**Pull request merged**